common/src/main/java/com/wynntils/models/containers/type/wynncontainers/AbilityTreeContainer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.containers.type.wynncontainers;
+
+import com.wynntils.models.containers.type.ScrollableContainerProperty;
+import com.wynntils.models.containers.type.WynncraftContainer;
+import java.util.regex.Pattern;
+
+public class AbilityTreeContainer extends WynncraftContainer implements ScrollableContainerProperty {
+    private static final Pattern NEXT_PAGE_PATTERN = Pattern.compile("§7Next Page");
+    private static final Pattern PREVIOUS_PAGE_PATTERN = Pattern.compile("§7Previous Page");
+
+    public AbilityTreeContainer() {
+        super(Pattern.compile("(?:Warrior|Shaman|Mage|Assassin|Archer) Abilities"));
```

Comment:
Although this specific case may not have a test at the moment, can we change the constructor pattern to extract the pattern into a static final field? Otherwise, these regexes become un-testable by our framework. Please change every container class.